stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/FileAttachmentsView.kt

```diff
@@ -194,8 +188,11 @@ private class FileAttachmentViewHolder(
     }
 
     private fun subscribeForProgressIfNeeded(attachment: Attachment) {
-        if (attachment.uploadState is Attachment.UploadState.InProgress) {
-            handleInProgressAttachment(binding.fileSize)
+        if (attachment.uploadState is Attachment.UploadState.Idle) {
+            handleInProgressAttachment(binding.fileSize, 0L, attachment.upload?.length() ?: 0)
+        } else if (attachment.uploadState is Attachment.UploadState.InProgress) {
+            val uploadState = attachment.uploadState as Attachment.UploadState.InProgress
+            handleInProgressAttachment(binding.fileSize, uploadState.bytesRead, uploadState.totalBytes)
```

Comment:
I think this is a bit cleaner for the same thing:

```suggestion
        val uploadState = attachment.uploadState
        if (uploadState is Attachment.UploadState.Idle) {
            handleInProgressAttachment(binding.fileSize, 0L, attachment.upload?.length() ?: 0)
        } else if (uploadState is Attachment.UploadState.InProgress) {
            handleInProgressAttachment(binding.fileSize, uploadState.bytesRead, uploadState.totalBytes)
```